connect-example/src/main/java/com/stripe/android/connect/example/data/SettingsService.kt

```diff
@@ -91,7 +91,8 @@ class SettingsService @Inject constructor(@ApplicationContext context: Context)
             presentationStyleIsPush = !sharedPreferences.getBoolean(PRESENTATION_IS_MODAL, false),
             embedInTabBar = sharedPreferences.getBoolean(EMBED_IN_TABBAR, false),
             embedInNavBar = !sharedPreferences.getBoolean(DISABLE_EMBED_IN_NAVBAR, false),
-            useXmlViews = sharedPreferences.getBoolean(USE_XML_VIEWS, false)
+            useXmlViews = sharedPreferences.getBoolean(USE_XML_VIEWS, false),
+            edgeToEdgeEnabled = sharedPreferences.getBoolean(EDGE_TO_EDGE_ENABLED, false)
```

Comment:
nit: Could we rename this and change the copy to "Enable edge-to-edge"?
```suggestion
            enableEdgeToEdge = sharedPreferences.getBoolean(ENABLE_EDGE_TO_EDGE, false)
```
We should also change the other copy to "Use XML views" (sentence-case).